components/container/Tabs.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const toKey = (index, prefix = '') => `${prefix}${index}`;
+
+const Tabs = ({ panels = [], index = 0 }) => {
```

Comment:
What do you think about making it a controlled component by default?

We can add another `SimpleTabs` component that keeps state.